models/aeif_cbvg_2010.h

```diff
@@ -0,0 +1,430 @@
+#include "config.h"
+
+#ifdef HAVE_GSL
+
+// External includes:
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_matrix.h>
+#include <gsl/gsl_odeiv.h>
+
+// Includes from nestkernel:
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "recordables_map.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+namespace nest
+{
+/**
+ * Function computing right-hand side of ODE for GSL solver.
+ * @note Must be declared here so we can befriend it in class.
+ * @note Must have C-linkage for passing to GSL. Internally, it is
+ *       a first-class C++ function, but cannot be a member function
+ *       because of the C-linkage.
+ * @note No point in declaring it inline, since it is called
+ *       through a function pointer.
+ * @param void* Pointer to model neuron instance.
+ */
+extern "C" int aeif_cbvg_2010_dynamics( double, const double*, double*, void* );
+
+/** @BeginDocumentation
+Name: aeif_cbvg_2010 - Exponential integrate-and-fire neuron
+                        model according to Clopath et al. (2010).
+
+Description:
+
+aeif_cbvg_2010 is an implementation of the neuron model as it is used in [1].
+It is an extension of the aeif_psc_delta model and capable of connecting to a
+Clopath synapse.
+
+Note that there are two points that are not mentioned in the paper but
+present in a Matlab implementation by Claudia Clopath. The first one is the
+clamping of the membrane potential to a fixed value after a spike occured to
+mimik a real spike and not just the upswing. This is important since the finite
+duration of the spike influences the evolution of the convolved versions
+(u_bar_plus/minus) of the membrane potential and thus the change of the
+synaptic weight. Secondly, there is a delay with which u_bar_plus/minus are
+used to compute the change of the synaptic weight.
+
+Parameters:
+
+The following parameters can be set in the status dictionary.
+
+Dynamic state variables:
+V_m         double - Membrane potential in mV.
+w           double - Spike-adaptation current in pA.
+z           double - Spike-adaptation current in pA.
+V_T         double - Spike-adaptation current in pA.
+
+Membrane Parameters:
+C_m         double - Capacity of the membrane in pF
+t_ref       double - Duration of refractory period in ms.
+V_reset     double - Reset value for V_m after a spike. In mV.
+E_L         double - Leak reversal potential in mV.
+g_L         double - Leak conductance in nS.
+I_e         double - Constant external input current in pA.
+tau_plus    double - Time constant of u_bar_plus.
+tau_minus   double - Time constant of u_bar_minus.
+tau_bar_bar double - Time constant of u_bar_bar.
+
+Spike adaptation parameters:
+a          double - Subthreshold adaptation in nS.
+b          double - Spike-triggered adaptation in pA.
+Delta_T    double - Slope factor in mV.
+tau_w      double - Adaptation time constant in ms.
+V_t        double - Spike initiation threshold in mV
```

Comment:
This line is of course wrong, however I think it is a remainder of `V_th`, which is not present in the model.  `V_T` is actually already there (a little bit further up). We removed this line from the documentation.

Reply:
I am sorry, while working on the `Delta_T` issue, I realized what your question was really about. Yes, we chose `V_T` instead of `V_th` to stay closer to the notation of the paper. However, in addition our `V_T` is a adaptive threshold (that is thus located in the Dynamic state variables section of the variables). Now (0ce9376) I also added `V_T_max` and `V_T_rest` to the documentation, which determine the adaptive behavior of `V_T`. (In general a quick search revealed that both `V_T` and `V_th` are used as spike initiation variable in various models)

Reply:
Changed the name of the variable to `V_th` with 511fe48. Now this should be resolved completely.